evento/app/controllers/users_controller.rb

```diff
@@ -63,7 +64,11 @@ def set_user
   end
 
   # Only allow a trusted parameter "white list" through.
-  def user_params
-    params.require(:user).permit(:name, :email, :password)
+  def create_params
```

Comment:
I applied the discussed updates in 252359e, but did not add validity checks for request body